src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -272,6 +272,124 @@ const listifyCenters = (center, idx, list) => {
   );
 };
 
+export const documentTypeEnums = [
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9',
+  '10',
+  '11',
+  '12',
+  '13',
+  '14',
+  '15',
+  '16',
+  '17',
+  '18',
+  '19',
+  '20',
+  '21',
+  '22',
+  '23',
+  '24',
+  '25',
+  '26',
+  '27',
+  '28',
+  '29',
+  '30',
+  '31',
+  '32',
+  '33',
+  '34',
+  '35',
+  '36',
+  '37'
+];
+
+export const documentTypes = [
+  'Application for Automotive Adaptive Equipment',
+  'Application for Compensation for Unemployability',
+  'Application for Specially Adapted Housing',
+  'Examination for Housebound Status',
+  'Request for Nursing Home Information',
+  'Examination for Housebound Status',
+  'Temporary Total Disability for PTSD_21_0781 and PTSD_21_0781A',
+  'Service Connected PTSD Statement',
+  'Service Connected Personal Assault PTSD Statement',
+  'Police Report',
+  'Military Performance Report',
+  'ER Medical Report',
+  'Private Medical Treatment Record',
```

Comment:
@egarreau @djmunoz1 it looks like some of these document types such as 'ER Medical Report' and 'Private Medical Treatment Record' actually have the same VBMS eDocument Document Type? since that's the case it seems like we will actually want to keep the current enum scheme so that these are mapped to the appropriate label?

Reply:
We'll need to ask EVSS about this; the backend isn't currently set up to handle two document types with the same VBMS eDocument Document Type. We'll need to gather more information on what the scenario is and how we should handle it.